yksgi/src/lib.rs

```diff
@@ -278,7 +278,7 @@ impl SGInterp {
             // If the current frame has pushed RBP we need to do the same (unless we are processing
             // the bottom-most frame).
             if pinfo.hasfp && i > 0 {
-                rsp = unsafe { rsp.offset(-POINTER_ISIZE) };
+                rsp = unsafe { rsp.offset(-ISIZEOF_POINTER) };
```

Comment:
Isn't this `rsp.sub(USIZEOF_POINTER)` in disguise? [I tend to prefer avoiding negative `isize`s because we inevitably end up having usize <-> isize conversions and those are surprisingly easy to get wrong.]

Reply:
I didn't know there was a `sub` call. :man_facepalming: Fixed in cf3e4be3b4da5c85522c35649d24fd7e155dd76a.

Sadly this (in combination with add) doesn't allow us to get rid of the ISIZE_POINTER constant, since we still have some places where we get passed an isize offset.

Reply:
OK.